tools/justfile

```diff
@@ -0,0 +1,144 @@
+# Default values for the deployment
+tag    := "test"
+queue  := "active"
+worker := "htcmock"
+
+# Base compose file
+export COMPOSE_BASE := "../docker-compose/docker-compose.yml"
+
+# loggin level of deployment
+export LOGGING_LEVEL:="Information"
+
+# Sets the queue
+export QUEUE := if queue == "rabbit" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if  queue == "rabbit091" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "artemis" {
+  "../docker-compose/docker-compose.queue-artemis.yml"
+}else{
+  "../docker-compose/docker-compose.queue-activemqp.yml"
+}
+
+
+# Sets the override to feed docker-compose
+export OVERRIDE := if  queue == "rabbit091" {
+  "../docker-compose/docker-compose.override-rabbitmq091.yml"
+}else{
+  "../docker-compose/docker-compose.override.yml"
+}
+
+# Defines worker and enviroment variables for deployment
+export ARMONIK_WORKER:= if worker == "stream" {
```

Comment:
Be careful with syntax. It is a bit weird here, and inconsistent

Reply:
Which syntax exactly? the `foo := if condition { value_true } {  value_false }`  is the provided in `just` documentation

Reply:
No, I mean the inconsistency with the spaces around the assignment operator. (and around `else`, also)

Reply:
I see, ok i'll take care of it.